zerver/lib/markdown/api_arguments_table_generator.py

```diff
@@ -91,7 +91,10 @@ def render_table(self, arguments: Sequence[Mapping[str, Any]]) -> List[str]:
         # TODO: Fix naming now that this no longer renders a table.
         table = []
         argument_template = """
-<div class="api-argument">
+<div>
+    <a href="#{argument}"></a>
+</div>
```

Comment:
I think adding the id attribute is sufficient and these three lines don't add anything. You can check on one of the localhost:9991/api pages.

Reply:
@orientor I have checked you are right these three lines not adding anything, so I have to do git commit --amend and then push to same branch after making changes right?

Reply:
Yes `git commit --amend` and then `git push --force`.